Mathlib/SetTheory/Cardinal/Aleph.lean

```diff
@@ -361,6 +371,30 @@ theorem aleph0_lt_aleph_one : ℵ₀ < ℵ₁ := by
 theorem countable_iff_lt_aleph_one {α : Type*} (s : Set α) : s.Countable ↔ #s < ℵ₁ := by
   rw [← succ_aleph0, lt_succ_iff, le_aleph0_iff_set_countable]
 
+@[simp]
+theorem aleph1_le_lift {c : Cardinal.{u}} : ℵ₁ ≤ lift.{v} c ↔ ℵ₁ ≤ c := by
+  rw [← Ordinal.lift_one.{u, v}, ← lift_aleph, lift_le]
```

Comment:
Can you make all these proofs more robust by doing
```suggestion
  simpa using lift_le (a := ℵ₁) (b := c)
```
and similar below? You might not have to specify `b`